src/crepe/api/Transform.h

```diff
@@ -32,11 +103,11 @@ class Transform : public Component {
 
 public:
 	//! Translation (shift)
-	Point position;
+	Vector2 position = {0,0};
 	//! Rotation, in radians
-	double rotation;
+	double rotation = 0;
```

Comment:
Het is niet de bedoeling om hier (in de .h file) default waardes toe te kennen aan rotation omdat in de constructor deze waardes direct overschreven worden.